apiserver/facades/client/application/application_unit_test.go

```diff
@@ -829,9 +829,9 @@ func (s *ApplicationSuite) TestDestroyApplication(c *gc.C) {
 	results, err := s.api.DestroyApplication(params.DestroyApplicationsParams{
 		Applications: []params.DestroyApplicationParams{{ApplicationTag: "application-postgresql"}},
 	})
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(results.Results, gc.HasLen, 1)
-	c.Assert(results.Results[0], jc.DeepEquals, params.DestroyApplicationResult{
+	c.Check(err, jc.ErrorIsNil)
+	c.Check(results.Results, gc.HasLen, 1)
+	c.Check(results.Results[0], jc.DeepEquals, params.DestroyApplicationResult{
```

Comment:
A blanket replace of Assert for Check is not reasonable. In the above scenario, if the error is nil, you'd panic on the last Check.

In this case it should be:
```
c.Assert(err, jc.ErrorIsNil)
c.Assert(results.Results, gc.HasLen, 1)
c.Check(results.Results[0], jc.DeepEquals, params.DestroyApplicationResult{
```

Check is for values, Assert is for errors or things that will lead to panics later.